desktop/cmp/contextmenu/ContextMenuSupport.js

```diff
@@ -0,0 +1,53 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2018 Extremely Heavy Industries Inc.
+ */
+
+import {defaultMethods} from '@xh/hoist/utils/ClassUtils';
+import {contextMenu} from './ContextMenu';
+import {ContextMenuTarget} from '@xh/hoist/kit/blueprint';
+
+/**
+ * Mixin to allow a Component to show a ContextMenu.
+ *
+ * Components that use this mixin should implement a getContextMenuItems()
+ * method.
+ *
+ * See Blueprint documentation for more information about the implementation of this
+ * behavior.
+ */
+export function ContextMenuSupport(C) {
+    C.isContextMenuSupport = true;
```

Comment:
`hasContextMenuSupport` more natural (vs "is")?

Reply:
yea, agree, 'has' would be more natural, but trying to balance that against the convention that
we put an 'isMixinName' on all the mixins.

I guess it points to the convention of 'xxxSupport' not being great.  We could call it 'ContextMenuHost' if you prefer? Want to stay from 'ContextMenuTarget' in this case to avoid confusion with Blueprint.

Reply:
'ContextMenuSource' also possible